rotkehlchen/db/filtering.py

```diff
@@ -787,3 +793,12 @@ def prepare(self) -> Tuple[List[str], List[Any]]:
         filters = [f'counterparty IN ({questionmarks})']
         bindings = self.protocols
         return filters, bindings
+
+
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBIgnoredAssetsFilter(DBFilter):
```

Comment:
add a docstring for this. It's kind of weird to have an entire filter only for this I guess.

Reply:
I will add a docstring but I don't have any other ideas about approaches in mind right now. Do you?

Reply:
No I don't. So go for it but we may optimize in the future.